### DISCUSSION TITLE
ABL version with BLT

### DISCUSSION THREAD

**Discussion by lin-ycv** [mriscoc/Ender3V2S1#1376] [Ideas]
Can a ABL version with BLT be released? I want to use Adaptive Bed Mesh in my slicer, but UBL doesn't support ABM.

**Comment by mriscoc** [comment]
AML can be used with UBL and ABL, all the firmwares posted here without the UBL letters in their filename are ABL.

**Reply by lin-ycv** [reply]
I dont think UBL supports ABM, some of the parameters ABM uses are for other features in UBL

Running ABM code on UBL causes the printer to remesh the whole building plate, instead of just the print area.

https://github.com/SoftFever/OrcaSlicer/wiki/adaptive-bed-mesh
```
; Marlin don't support speicify the probe count yet, so we only specify the probe area
G29 L{adaptive_bed_mesh_min[0]} R{adaptive_bed_mesh_max[0]} F{adaptive_bed_mesh_min[1]} B{adaptive_bed_mesh_max[1]} T V4
```

In the release section, I don't see a firmware for Ender3v2 with BLT but without UBL

**Reply by mriscoc** [reply]
ABM != **[AML](https://github.com/mriscoc/Ender3V2S1/wiki/Adaptive-Mesh-Leveling-(AML))**, In the professional firmware AML support automatic mesh density and mesh area. If you want a Ender3v2 with BLT but without UBL firmware you can compile without mark the UBL check box in the Configurator: https://github.com/mriscoc/Special_Configurations#creating-your-own-custom-configuration
